src/server.js

```diff
@@ -491,11 +437,17 @@ function performAuthentication(data) {
 	}
 
 	// Perform password checking
+	let auth = function() {};
```

Comment:
Just do `let auth;`, there is no need to assign anything.

Reply:
I'm not so fan of a variable switching its type as the program goes. :/

Reply:
yay for javascript /s

realistically you're doing an unnecessary allocation here that gets thrown away.